sscep/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+SSCEP=sscep-0.9.0
+
+b_log "Building sscep"
+PREFIX_SSCEP=${TOPDIR}/phoenix-rtos-ports/sscep
+PREFIX_SSCEP_BUILD=${PREFIX_BUILD}/sscep
+PREFIX_SSCEP_SRC="${PREFIX_SSCEP_BUILD}/${SSCEP}"
+PREFIX_SSCEP_MARKERS="${PREFIX_SSCEP_BUILD}/markers"
+PREFIX_SSCEP_INSTALL="${PREFIX_SSCEP_BUILD}/install"
+
+#
+# Download and unpack
+#
+mkdir -p "$PREFIX_SSCEP_BUILD"
+
+if [ -n "$CLEAN" ]; then
```

Comment:
this is no longer needed as build dir is inside _build (it will be cleaned globally and we don't support cleaning single components from ports for now